### PR TITLE
Change temporary group name (remove underscore) and change ui message

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by pmauduit** [comment]
The only remark is that the message in the `alert()` cannot be translated (easily) ? IIRC there are some other places in the UI where the strings are not translated.

Looks good !

**Comment by Vampouille** [comment]
If we want to provide translation for messages in js files, I think we need to move all messages to a jsp file.

File: ldapadmin/src/main/filtered-resources/WEB-INF/views/privateUi.jsp [hunk 1]
```diff
@@ -110,6 +111,7 @@ try {
     var GEOR_config = {
         publicContextPath: "<%= contextPath %>"
     };
+    var VIRTUAL_TEMPORARY_GROUP_NAME = "<%= GroupsController.VIRTUAL_TEMPORARY_GROUP_NAME %>";
```
Comment (fvanderbiest) [inline ldapadmin/src/main/filtered-resources/WEB-INF/views/privateUi.jsp]: Better use the above config object instead:
```js
GEOR_config = {
          publicContextPath: "<%= contextPath %>",
          virtualTempGroupName: <%= GroupsController.VIRTUAL_TEMPORARY_GROUP_NAME %>
};
```

File: ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/groups/GroupsController.java [hunk 2]
```diff
@@ -55,7 +55,7 @@
 
 	private static final String USER_NOT_FOUND = "user_not_found";
 
-	private static final String VIRTUAL_TEMPORARY_GROUP_NAME = "TEMPORARY_USER";
+	public static final String VIRTUAL_TEMPORARY_GROUP_NAME = "TEMPORARY-USER";
```
Comment (fvanderbiest) [inline ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/groups/GroupsController.java]: Or just "TEMPORARY" ?

**Comment by fvanderbiest** [comment]
> If we want to provide translation for messages in js files, I think we need to move all messages to a jsp file.

ldapadmin is not i18n right now. This will be adressed in a second time. 
So, for me, your contribution is fine regarding the error message.

**Comment by fvanderbiest** [comment]
You merged too soon !

**Comment by Vampouille** [comment]
I just commit on master (5b178320d758073dbca768a1f429c685de15545c) :
  * move VIRTUAL_TEMPORARY_GROUP_NAME var in GEOR_config object
  * rename group to : Temporary

**Pull request merged**